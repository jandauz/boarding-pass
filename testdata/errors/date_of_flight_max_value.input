M1DESMARAIS/LUC       EABC123 YULFRAAC 0834 367J001A0025 100