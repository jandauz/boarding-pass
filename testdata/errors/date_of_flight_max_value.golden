bcbp: Invalid data format:
  boarding pass data:
  | "M1DESMARAIS/LUC       EABC123 YULFRAAC 0834 367J001A0025 100"
  |                                              ^ got "367"
  |
  = reason: data for "Date of Flight (Julian Date)" must be 3 digits with leading zeroes with maximum value of 365 (366 for leap years)
